physics/module_sf_ruclsm.F90

```diff
@@ -19,28 +19,53 @@ MODULE module_sf_ruclsm
    public :: lsmruc, ruclsminit, rslf
 
 !> CONSTANT PARAMETERS
+!! @{
       real (kind=kind_phys), parameter :: P1000mb = 100000.
```

Comment:
Most of these are already defined in [physcons.F90](https://github.com/ufs-community/ccpp-physics/blob/ufs/dev/physics/physcons.F90). It's ideal to pass these into the schemes directly, or at least pass the constants to the _init() phase and set internal module variables that are used throughout the scheme.

Reply:
The change is done to use constants from physcons.F90.